packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -566,6 +566,35 @@ contract LockedGold is
     return _updateDelegatedAmount(delegatorAccount, delegateeAccount);
   }
 
+  /**
+   * @notice Returns the pending withdrawals from unlocked gold for an account in a given range.
+   * @param account The address of the account.
+   * @param from The start index of the pending withdrawals.
+   * @param to The end index of the pending withdrawals.
+   * @return The value for each pending withdrawal.
+   * @return The timestamp for each pending withdrawal.
+   */
+  function getPendingWithdrawalsInBatch(address account, uint256 from, uint256 to)
```

Comment:
Do we have a convenience view function returning the length of an account's array? If not, worth adding.

Reply:
We have